spec/models/category_spec.rb

```diff
@@ -13,28 +13,14 @@
 #
 
 describe Category do
-  context 'creation' do
-    it 'is valid' do
-      category = create(:category, :with_user)
-      expect(category).to be_valid
-      expect(Category.count).to eq(1)
-    end
+  it { is_expected.to respond_to :friendly_id }
 
-    it 'is invalid without a user_id' do
-      category = build(:category, user_id: nil)
-      expect(category).to be_invalid
-    end
-
-    it 'is invalid without a name' do
-      category = build(:category, name: nil)
-      expect(category).to have(1).error_on(:name)
-    end
```

Comment:
Could you explain why you're removing these error relayed tests?

Reply:
Hi Julia! Since I'm validating presence with the `validate_presence_of` matcher, for me this spec is redundant. In a matter of fact, we'll have more redundant specs in other models, but I'm not sure what should we do in this case.

If we keep them, our suite will grow bigger, but we'll have redundant specs. To remove them, we'd need to be sure that we're still testing each case.

For now, I'll keep this test under the `validations` context, and will start a discussion in the dev channel to get some insights, what do you think?

Reply:
Thanks for explaining! Sounds good to me to remove!